js/views/settingsVw.js

```diff
@@ -758,6 +787,7 @@ module.exports = Backbone.View.extend({
     if(newAddress.name || newAddress.street || newAddress.city || newAddress.state || newAddress.postal_code) {
       if(!newAddress.name || !newAddress.street || !newAddress.city || !newAddress.state || !newAddress.postal_code){
         messageModal.show(window.polyglot.t('errorMessages.saveError'), window.polyglot.t('errorMessages.missingError'));
+        setTimeout(() => $(e.target).removeClass('loading'), 0);
```

Comment:
I'm kind of leaning towards having each view set the loading class, mostly because originally I thought there would be more places it would be needed. 

A small loading button plugin may be good too, if we decide we want to put more functionality into the loading buttons (like error states, or different messages depending on what happens with the loading action).